challenges/challenges.py

```diff
@@ -6,18 +6,14 @@
 import asyncio
 from datetime import datetime
 
+
 class Challenges(commands.Cog):
-    
+
     def __init__(self, bot):
         self.bot = bot
-        self.config = Config.get_conf(self, identifier=25202025)
-        default_member = {'tokens' : 0, 'tracking' : False, 'lastBattleTime' : "20200627T170000.000Z", 'progress' : 0, 'plant' : None, 'wins' : {}, 'loses' : {}}
+        self.config = Config.get_conf(self, identifier=424242696942)
+        default_member = {'tracking': False, 'lastBattleTime': "20200627T170000.000Z", 'entries': 0, 'streak': 0}
```

Comment:
Not sure whether I should add win/lose purely for stats, but I feel like it'll be kinda more boring than with pvz.